lib/workers/global/config/parse/env.ts

```diff
@@ -237,5 +238,28 @@ export async function getConfig(
 
   unsupportedEnv.forEach((val) => delete env[val]);
 
+  config = migratePlatformOptions(config, env);
+  return config;
+}
+
+function migratePlatformOptions(
+  config: AllConfig,
+  env: NodeJS.ProcessEnv,
+): AllConfig {
+  const platformOptionsKeys = ['platformVersion', 'gitLabIgnoreApprovals'];
+  const platformOptions: Record<string, unknown> = {};
+  let updated = false;
+  for (const key of platformOptionsKeys) {
+    const envKey = getEnvName({ name: key });
+    if (!is.undefined(env[envKey])) {
+      updated = true;
+      platformOptions[key] = env[envKey];
+    }
+  }
+
+  if (updated) {
```

Comment:
```suggestion
  if (is.nonEmptyObject(platformOptions)) {
```
why not using same code here too